recipes/nbtutor/build.sh

```diff
@@ -0,0 +1,19 @@
+conda install --yes nodejs
+# FIXME: This is an ugly hack. nodejs only gets installed and linked in the
+# first build env and doesn't get linked in following build envs
```

Comment:
#2067 had reported a similar issue with `libpng`. They also reported that building locally with an updated `conda-build` got rid of the issue and all builds passed. I have not yet tried this, I can if you would like?